
package it.kdm.docer.core.authentication;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.7.10
 * 2021-07-15T14:46:50.642+02:00
 * Generated source version: 2.7.10
 */

@WebFault(name = "AuthenticationServiceException", targetNamespace = "http://authentication.core.docer.kdm.it")
public class AuthenticationServiceException_Exception extends java.lang.Exception {
    
    private it.kdm.docer.core.authentication.AuthenticationServiceException authenticationServiceException;

    public AuthenticationServiceException_Exception() {
        super();
    }
    
    public AuthenticationServiceException_Exception(String message) {
        super(message);
    }
    
    public AuthenticationServiceException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public AuthenticationServiceException_Exception(String message, it.kdm.docer.core.authentication.AuthenticationServiceException authenticationServiceException) {
        super(message);
        this.authenticationServiceException = authenticationServiceException;
    }

    public AuthenticationServiceException_Exception(String message, it.kdm.docer.core.authentication.AuthenticationServiceException authenticationServiceException, Throwable cause) {
        super(message, cause);
        this.authenticationServiceException = authenticationServiceException;
    }

    public it.kdm.docer.core.authentication.AuthenticationServiceException getFaultInfo() {
        return this.authenticationServiceException;
    }
}
